### ISSUE TITLE
Could not select any API entry - 3005 - Internal error

### ISSUE THREAD

**Issue by dzonelabuser** [wazuh/wazuh-kibana-app#2477]
Wazuh: 3.13
Kibana version: 7.9.0
Elasticsearch version: 7.9.0
Browser version: Chrome

**Description**
This is a new installation and the first time working with this software. When I go to the kibana app and try to connect to my Wazuh Api, I get the error "coult not select and API entry, then 3005 - internal error". 

The username/password credentials works fine without any issue. So I have ruled that out; as well as the protocol. 

I am unable to use internet explorer to bring up the kibana app UI. I do not have firefox installed on my server. The only browser somewhat working is Chrome. 

I brought up the developer tools selected Network then XHR in the console area I see the errors

"Refused to execute inline script because it violates the following Content Security Policy directive: "script-src wazuh#/settings?_g=()&tab=api:342 'unsafe-eval' 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-P5polb1UreUSOe5V/Pv7tc+yeZuJXiOi/3fqhGsU7BE='), or a nonce ('nonce-...') is required to enable inline execution." 

"A single error about an inline script not firing due to content security policy is expected!
kbn-ui-shared-deps.js:453 INFO: 2020-09-21T13:22:45Z
  Adding connection to http://xx.xxx.xx.xxx:5601/elasticsearch"

Failed to load resource: the server responded with a status of 500 (Internal Server Error) http://xx.xxx.xx.xxx:5601/api/check-api

Failed to load resource: the server responded with a status of 500 (Internal Server Error) http://xx.xxx.xx.xxx:5601/api/check-api
Error set extensions							                        wazuh.bundle.js:2
4001 - Cannot set property 'extensions' of undefined			wazuh.bundle.js:2
Detected an unhandled Promise rejection.				        
<img width="613" alt="API-Connection Issue" src="https://user-images.githubusercontent.com/61987824/93787091-0dcc9e00-fbfe-11ea-82de-e79c309ac416.PNG">
<img width="539" alt="API-Connection Issue-1" src="https://user-images.githubusercontent.com/61987824/93787093-0dcc9e00-fbfe-11ea-85ec-4c354df17b95.PNG">
![XHR-Screenshot](https://user-images.githubusercontent.com/61987824/93787094-0e653480-fbfe-11ea-80eb-e23c0ee6191d.png)

wazuh.bundle.js:2
4001 - Cannot set property 'extensions' of undefined

"Uncaught (in promise) 4001 - Cannot set property 'extensions' of undefined		wazuh.bundle.js:2 
asyncGeneratorStep @ wazuh.bundle.js:2
_throw @ wazuh.bundle.js:2"

**Additional context**
I am running the setup on two different systems both running centOS 7

Server 1: Running the Wazuh Manager, Wazuh Api and Filebeat 
Server 2: Running Elasticsearch, Kibana and Wazuh App

The only thing I've done with this machine is Installed the software, get the API working, deployed three clients (1 centos 7 and 2 Windows) .

**Please can someone assist me as I have been trying to resolve these issues without no success and I have a timeline with this project.**

**Comment by jsanchez91** [comment]
Hi @dzonelabuser 

In order to help you, you could indicate me the error message that appears when you hover over the icon `?`.

![image](https://user-images.githubusercontent.com/3064506/93876570-586e1900-fcd7-11ea-884e-bf1c477cd66d.png)

**Comment by dzonelabuser** [comment]
> Hi @dzonelabuser
> 
> In order to help you, you could indicate me the error message that appears when you hover over the icon `?`.
> 
> ![image](https://user-images.githubusercontent.com/3064506/93876570-586e1900-fcd7-11ea-884e-bf1c477cd66d.png)

**Hi  jsanchez91,** 

When I hover the question mark, the error that appears is "3005 - Internal error" But then also this message "Could not select any API entry" appears & disappears after refreshing the browser. 

Last night I did a tail on the wazupp app server and also got the below output
"level":"error","location":"wazuh-api:setExtensions","message":"Cannot set property 'extensions' of undefined"}
"level":"error","location":"wazuh-api:checkAPI","message":"Internal error"}
"level":"error","location":"update-registry:updateAPIHostname","message":"Cannot set property 'extensions' of undefined"}

**Comment by jsanchez91** [comment]
Hi @dzonelabuser,

You may have a configuration error or a connectivity problem between your servers.

To resolve this problem, we will perform the following checks.

On your Kibana server (Server 2) check if the configuration within the wazuh.yml file is correct. It should look like this:

```yml
hosts:
  - default:
     url: https://<your_server_1_ip>
     port: 55000
     username: foo # Default user and password
     password: bar
```

Check if server 2 has access to the Wazuh API on server 1. 
```bash
curl -k -u foo:bar https://<your_server_1_ip>:55000
```

**Comment by dzonelabuser** [comment]
I resolved the issue. 
For some reason, I couldn't even use yum remove wazuh-manager to uninstall wazuh-manager. 
It kept giving me package not found. 

**RESOLUTION**

1. Stopped wazuh-manager, wazuh-api and filebeat

2. Removed filebeat

3. Executed the below: 
/var/ossec/bin/ossec-control stop && rm -rf /var/ossec/ && rm -f /etc/init.d/*ossec* && rm -f /etc/ossec-init.conf

4. Reinstalled wazuh-manager 3.13.2 and wazuh-api 3.13.2

5. Also reinstalled filebeat

6. Went through all the API configurations

Basically after performing these steps, I was able to get wazuh app to function and add agents without any issues. 


Thank you very much @jsanchez91
